### DISCUSSION TITLE
Shell history: disable relevance/score sorting by default (as if hit C-R a second time)

### DISCUSSION THREAD

**Discussion by jlee1-made** [junegunn/fzf#2899] [Q&A] [answered]
When I use `C-R` to search backwards through my shell history, initially it is sorted by relevance ("score").  If I then hit `C-R` a second time, it's sorted by order of the commands in history.

I want that first `C-R` to default to sorting by command order, not by relevance.

How can I do that?

**Comment by HbombHn** [comment]
I was wondering why my fzf behaved differently on my old laptop, and came to ask something similar to this, I didn't know you can change that by hit ctrl+r again! Saw the inline info I had it changed from `+S` to `-S`. So I looked through the help output and I think the option is 
`    +s, --no-sort         Do not sort the result`
So I did a `export FZF_DEFAULT_OPTS='+s'` and that seems to make it the preferred method when searching command history which is mint!!

**Reply by junegunn** [reply]
You should set `FZF_CTRL_R_OPTS` instead of `FZF_DEFAULT_OPTS` because you want to disable sorting only for CTRL-R.

**Comment by junegunn** [comment, marked as answer]
You might want to read through https://github.com/junegunn/fzf#key-bindings-for-command-line

```sh
export FZF_CTRL_R_OPTS='--no-sort'
```
